This is a new 4rd part of student management system ,with emails

aharutyunyan9 opened the pull request:
About this requirement "In written form, send your advice on how to use functions in our code"

I think here I can use a function to get_positive_number ,which  could be used for age and grades  inputs

aharutyunyan9 committed:
This is 4rd part

aharutyunyan9 committed:
Merge pull request #1 from aharutyunyan9/Student-managment-system

Student's management system 3

Yana-Gabrielyan commented on students 4 with emails.py:
```diff
@@ -0,0 +1,111 @@
+    # Get student's age
+    age_in = input("Enter student's age: ")
+    if not age_in.isdigit():
+        retry = input("Invalid age! Digits only: ")
```
Only retries once — could improve with a while loop

Yana-Gabrielyan commented on students 4 with emails.py:
```diff
@@ -0,0 +1,111 @@
+        print(" You are a College student.")
+
+    # Current year grade
+    grade_current = input("Enter grade for current year: ")
```
Use float() instead of int() for more realistic grades , the same for grade_past

Yana-Gabrielyan reviewed (approved):
Good job , one suggestion - you can start to use functions, could modularize the code (e.g., validate_input, generate_email, etc.)

aharutyunyan9 committed:
This is the 5th part of student management system 

The part with  importing from a file
Also sharing the files with import and export

aharutyunyan9 committed:
This is the part with functions and logging

aharutyunyan9 committed:
Merge pull request #3 from aharutyunyan9/Student-managment-system

This is the 5th part of student management system
